CHECK: expected string not found in input

https://dev.azure.com/dnceng-public/public/_build/results?buildId=333311&view=ms.vss-test-web.build-test-results-tab

Numerous failures like the following:
```
07:38:01.038 Running test: JIT/opt/Add/IntAdd/IntAdd.cmd
__tmp0_IntAdd.cs:12:11: error: CHECK: expected string not found in input
// CHECK: Lcl frame size =
          ^
__jit_disasm.out:1:86: note: scanning from here
; Assembly listing for method CodeGenTests.IntAdd:Int8_Add(byte,byte):byte (FullOpts)
                                                                                     ^
__jit_disasm.out:5:9: note: possible intended match here
; rsp based frame
        ^

Input file: __jit_disasm.out
Check file: __tmp0_IntAdd.cs

-dump-input=help explains the following input dump.

Input was:
<<<<<<
            1: ; Assembly listing for method CodeGenTests.IntAdd:Int8_Add(byte,byte):byte (FullOpts) 
check:12'0                                                                                          X error: no match found
            2: ; Emitting BLENDED_CODE for X64 with AVX512 - Windows 
check:12'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            3: ; FullOpts code 
check:12'0     ~~~~~~~~~~~~~~~~
            4: ; optimized code 
check:12'0     ~~~~~~~~~~~~~~~~~
            5: ; rsp based frame 
check:12'0     ~~~~~~~~~~~~~~~~~~
check:12'1             ?          possible intended match
            6: ; partially interruptible 
check:12'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~
            7: ; No PGO data 
check:12'0     ~~~~~~~~~~~~~~
            8:  
check:12'0     ~
            9: G_M000_IG01: ;; offset=0000H 
check:12'0     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/src/coreclr/tools/SuperFileCheck/Program.cs
+++ b/src/coreclr/tools/SuperFileCheck/Program.cs
@@ -392,7 +392,6 @@ namespace SuperFileCheck
 
             // Create anchors from the first prefix.
             var startAnchorText = $"// {checkPrefixes[0]}-LABEL: for method {methodName}";
-            var startInstrsAnchorText = $"// {checkPrefixes[0]}: Lcl frame size =";
             var endAnchorText = $"// {checkPrefixes[0]}: for method {methodName}";
 
             // Create temp source file based on the source text of the method.
@@ -406,7 +405,6 @@ namespace SuperFileCheck
                 tmpSrc.AppendLine(String.Empty);
             }
             tmpSrc.AppendLine(startAnchorText);
-            tmpSrc.AppendLine(startInstrsAnchorText);
             tmpSrc.AppendLine(TransformMethod(methodDecl, checkPrefixes));
             tmpSrc.AppendLine(endAnchorText);
 

